server/services/featureToggleService.ts

```diff
@@ -0,0 +1,15 @@
+import FeatureToggleStore from '../data/featureToggleStore/featureToggleStore'
+
+export default class FeatureToggleService {
+  constructor(private readonly featureToggleStore: FeatureToggleStore) {}
+
+  public async getFeatureToggle(prisonId: string, featureToggle: string): Promise<boolean> {
+    if (!prisonId) return false
```

Comment:
I wonder if we were to use this for other things, we'd want to be able to supply a default (might not always be false by default?)

Reply:
One to implement when we get to it though, not necessarily for this PR